### PR TITLE
What's New 96

### PR DESCRIPTION
**Pull request by mikehoffms**
**Rendered article for review:**
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools?branch=pr-en-us-1566
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-1566#focus-mode

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:287]: Need content for Focus Mode section of this Experimental Features article.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:117]: I'm not really sure these 2 lines make sense as bullets in a list. I propose making these 2 sentences in one paragraph and then moving it down below the image. It's not must-have information but it is nice-to-have since we're not sure if this will cause undue clutter for users.

To clarify what I'm proposing: I'd add "To learn more about how DevTools fetches and parses sourcemaps..." as the last sentence in the paragraph.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:139]:
```suggestion
*  If the **Sources&& tool is open and you then turn on the **Open source files in Visual Studio Code** experiment, setting breakpoints or opening files now opens the **Sources** tool, rather than directing you to Visual Studio Code or to documentation to configure the experiment correctly.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:137]:
```suggestion
In Microsoft Edge version 96, the **Open source files in Visual Studio Code** experiment now integrates better with the **Sources** tool.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (zoherghadyali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:143]:
```suggestion
*  As with earlier versions of Microsoft Edge, if the **Sources** tool isn't open in DevTools and you then turn on the **Open source files in Visual Studio Code** experiment, setting breakpoints or opening files from tools other than the **Sources tool** opens the file in Visual Studio Code.
```

**Review by zoherghadyali — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:39]: I added this link.  Please compare the present section to the linked section.
@hxlnt @zoherghadyali

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (codepo8) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:26]: It is vertical by default, but you can also turn it horizontal, so I guess removing it might help.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (codepo8) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:90]: Headless mode is a different thing than the screencast, I opt for removing the first sentence.

**Review by codepo8 — APPROVED** [review]
Pending two small things added as comments, LGTM.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:26]:
```suggestion
The Microsoft Edge DevTools team is experimenting with a new DevTools UI: **Focus Mode**. Focus Mode reduces distractions and clutter with a more modern and simplified layout. The new Activity Bar allows you to pin your favorite tools in a horizontal or vertical toolbar to use screen space more efficiently.
```

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:33]:
```suggestion
This UI is still in development and may change in future versions of Microsoft Edge.  We're eager to hear your feedback about this new DevTools UI.  Send us your feedback by tweeting [@EdgeDevTools](https://twitter.com/edgedevtools).  Or, with the **Focus Mode** experiment turned on, at the bottom of the **Activity Bar**, select **Help** (![the Help icon in the Activity Bar in Focus Mode](../../../media/help-icon-of-focus-mode.png)) > **Feedback** to show the **Send feedback** window.

```

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (hxlnt) [reply microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:39]: This link should be removed because this functionality (tool grouping) no longer exists.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (hxlnt) [reply microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:26]: Left requested revisions below.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:55]:
```suggestion
**Console** can be opened in the bottom panel (the **Drawer**) while another tool is open:

```

**Review by hxlnt — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:97]: Should the "Open in Issues" button be shown in the image below? I don't see it in the current image.

File: microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md
Comment (Reezaali) [inline microsoft-edge/devtools-guide-chromium/whats-new/2021/11/devtools.md:101]: Similar comment as above - I don't see the "Don't show again" button in the image above.

**Review by Reezaali — APPROVED** [review]
Left two comments on the images.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:29]: I don't think we picked this up from upstream.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (hxlnt) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:287]: Could we leave it as a quick one-sentence description (maybe from What's New 96) and create a new issue assigned to me to write this at a later time?

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (hxlnt) [inline microsoft-edge/devtools-guide-chromium/experimental-features/index.md:421]: I think this no longer exists since it's no longer an experimental feature.

**Review by hxlnt — CHANGES_REQUESTED** [review]

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (hxlnt) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:29]: (Just the "Enable CSS length" one)

**Review by hxlnt — APPROVED** [review]

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:421]: Removed line.

**Comment by hxlnt** [comment]
Added a few more comments above on the experimental features page, but will go ahead and approve so this PR isn't blocked on me. Thanks!

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:29]: Not sure if there's action suggested here.  Copied comment to a PR that's about this page.  Closing this comment.

File: microsoft-edge/devtools-guide-chromium/experimental-features/index.md
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/experimental-features/index.md:287]: Strange GitHub PR UI: the "Resolve conversation" button is in "Files changed" tab but not in the "Conversation" tab.

Added serviceable content here, will push.

**Pull request merged**
